### PR TITLE
Fix build gradle

### PR DESCRIPTION
**Pull request by CustomEntity**
### How should this be manually tested?

  - Step 1 :
  - Step 2 :
  - Step 3 :  
  ...

### CODE REVIEW

File: src/main/kotlin/io/kuzzle/sdk/protocol/Http.kt [hunk 1]
```diff
@@ -220,10 +220,10 @@ open class Http : AbstractProtocol {
                         }
                     }
                     this.header("content-type", "application/json")
-                    this.body = JsonSerializer.serialize(payload)
+                    this.setBody("JsonSerializer.serialize(payload)")
```
Comment (Shiranuit) [inline src/main/kotlin/io/kuzzle/sdk/protocol/Http.kt:223]:
```suggestion
                    this.setBody(JsonSerializer.serialize(payload))
```
This should not be stringified

**Review by Shiranuit — CHANGES_REQUESTED** [review]
Good job, some comment about the naming of the SDK
You changed the SDK name without from `sdk-jvm-x.x.x-without-dependencies.jar` to `sdk-jvm-.x.x.x.jar`
And `sdk-jvm-.x.x.x.jar` to `sdk-jvm-x.x.x-with-dependencies.jar`

Here you can see that  `sdk-jvm-.x.x.x.jar` was with dependencies and you changed it to be without dependencies, once released on maven people that were importing ``sdk-jvm-.x.x.x` in their `pom.xml` or `build.gradle` will have issues because they will have missing dependencies.
We should keep the same naming as before to avoid confusing the users when they will be upgrading their version of the SDK.

Also can you please update the compatibility matrix in the `README.md`

File: build.gradle.kts [hunk 2]
```diff
@@ -35,8 +33,8 @@ val pomDeveloperId = "kuzzleio"
 val pomDeveloperName = "kuzzle"
 
 group = "io.kuzzle.sdk"
-version = "1.3.1"
-val ktorVersion = "1.6.8"
+version = "1.3.2"
+val ktorVersion = "1.3.2-1.4.0-rc"
```
Comment (Shiranuit) [inline build.gradle.kts:37]: Why did you downgrade the ktorVersion from `1.6.8` to `1.3.2-1.4.0-rc` ?

**Comment by CustomEntity** [comment]
> Good job +1 some comment about the naming of the SDK You changed the SDK name without from `sdk-jvm-x.x.x-without-dependencies.jar` to `sdk-jvm-.x.x.x.jar` And `sdk-jvm-.x.x.x.jar` to `sdk-jvm-x.x.x-with-dependencies.jar`
> 
> Here you can see that `sdk-jvm-.x.x.x.jar` was with dependencies and you changed it to be without dependencies, once released on maven people that were importing `sdk-jvm-.x.x.x` in their `pom.xml` or `build.gradle` will have issues because they will have missing dependencies. We should keep the same naming as before to avoid confusing the users when they will be upgrading their version of the SDK.
> 
> Also can you please update the compatibility matrix in the `README.md`

FIxed in commit 7436193552a980bb2b476477d640bebe5ca5559f

**Review by Shiranuit — APPROVED** [review]

**Review by Aschen — COMMENTED** [review]
I have no experience with Gradle so if it's ok for @Shiranuit it's ok for me

**Pull request merged**